js/app.js

```diff
@@ -1,23 +1,76 @@
-// START JS CODE
+// Configure a view object, to hold all our functions for dynamic updates and article-related event handlers.
+var projectView = {};
 
+projectView.populateFilters = function() {
+  $('article').each(function() {
+    if (!$(this).hasClass('template')) {
+      var val = $(this).find('address a').text();
+      var optionTag = '<option value="' + val + '">' + val + '</option>';
+      $('#coder-filter').append(optionTag);
+      val = $(this).attr('data-category');
+      optionTag = '<option value="' + val + '">' + val + '</option>';
+      if ($('#category-filter option[value="' + val + '"]').length === 0) {
+        $('#category-filter').append(optionTag);
+      }
+    }
+  });
+};
 
-//DOM--READY FUNCTION
-$(function() {
-  $('.tab-content').hide();
-});
+projectView.handleDevFilter = function() {
+  $('#coder-filter').on('change', function() {
+    if ($(this).val()) {
+      $('article').hide();
+      $('article[data-coders="' + $(this).val() + '"]').fadeIn();
```

Comment:
If you change this to data-author your filter were work. In your toHtml method you use data-author.